docs/docs/getting-started/install.md

```diff
@@ -119,24 +119,110 @@ The following [resource providers need to be registered](https://learn.microsoft
 
 By default, Constellation tries to register these automatically if they haven't been registered before.
 
-You need the following permissions for your user account:
+To [automatically create the IAM configuration](../workflows/config.md#creating-an-iam-configuration) for Constellation, you need the following permissions:
```

Comment:
```suggestion
To [create the IAM configuration](../workflows/config.md#creating-an-iam-configuration) for Constellation, you need the following permissions:
```
Nils agreed to revert this https://github.com/edgelesssys/constellation/pull/1442#discussion_r1147527831